packages/hooks/src/useExistingCFDAccounts.ts

```diff
@@ -56,8 +56,9 @@ const useExistingCFDAccounts = () => {
 
     return {
         data,
-        isLoading: [mt5_rest.isLoading, dxtrade_rest.isLoading, derivez_rest.isLoading].some(Boolean),
-        isSuccess: [mt5_rest.isSuccess, dxtrade_rest.isSuccess, derivez_rest.isSuccess].every(Boolean),
+        ...{ ...mt5_rest, ...dxtrade_rest, ...derivez_rest },
+        isLoading: mt5_rest.isLoading || dxtrade_rest.isLoading || derivez_rest.isLoading,
```

Comment:
@thisyahlen-deriv isn't i`sLoading = !isSuccess` ? why do we need to have them both?

Reply:
Will remove it since im not using it as well